code/mob/living/life/radiation.dm

```diff
@@ -0,0 +1,13 @@
+/datum/lifeprocess/radiation
+	process(datum/gas_mixture/environment)
+		if(!owner || !owner.radiation_dose)
+			return
+
+		//apply effects
+		if(!owner.hasStatus("radiation"))
+			owner.changeStatus("radiation",null)
```

Comment:
this is confusing, what is the purpose of this

Reply:
I'll add code comments to clear this up, but essentially it just gives the owner an infinite duration radiation status effect, assuming their dose is greater than 0 (early return checks for this above)